cmd/goal/ledger.go

```diff
@@ -17,13 +17,30 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"strconv"
 
 	"github.com/spf13/cobra"
+
+	"github.com/algorand/go-algorand/protocol/transcode"
+)
+
+var (
+	rawFilename    string
+	rawBlock       bool
+	base32Encoding bool
+	strictJSON     bool
 )
 
 func init() {
 	ledgerCmd.AddCommand(supplyCmd)
+	ledgerCmd.AddCommand(blockCmd)
+
+	blockCmd.Flags().StringVarP(&rawFilename, "out", "o", stdoutFilenameValue, "The filename to dump the raw block to (if not set, use stdout)")
+	blockCmd.Flags().BoolVarP(&rawBlock, "raw", "r", false, "Dump raw block as msgpack")
```

Comment:
nit: Change the description to "Format block as msgpack".
The fact that we're internally storing the block in it's msgpack encoded form isn't visible to the goal end-user.

Reply:
done!